pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -78,7 +86,18 @@ func (h *HeartBeat) heartBeat(heartBeatInterval time.Duration, stopCh chan struc
 	// So the amount of time between heartbeats randomly varies between 1min and 2min12sec + the heartbeat function execution time.
 	wait.JitterUntil(h.do, heartBeatInterval, 1.2, true, stopCh)
 
-	wait.JitterUntil(h.handleKSM, 10*time.Minute, 0.2, true, stopCh)
+	wait.JitterUntil(func() {
+		for err := h.handleKSM(); err != nil; {
+			select {
+			case <-stopCh:
+				return
+			default:
+				log.Log.Reason(err).Errorf("failed to set KSM label on node %s, trying again in 1 minute", h.host)
+				time.Sleep(1 * time.Minute)
+				err = h.handleKSM()
```

Comment:
nit: `for err := h.handleKSM(); err != nil; err =  h.handleKSM();{`

Reply:
done